vtr_flow/scripts/vtr-flow.py

```diff
@@ -203,6 +226,23 @@ def vtr_command_main(arg_list, prog=None):
                                    verbose=True if args.verbosity > 2 else False,
                                    echo_cmd=True if args.verbosity >= 4 else False)
     exit_status = 0
+    flow_type = 2 #Use iterative black-boxing flow for multi-clock circuits
+    use_new_latches_restoration_script = 1 
```

Comment:
It was agreed to enable it by default given not enabling gives the wrong circuit.
I don't think we should still provide a way to disable given that its "broken" without it, and doesn't affect the singly clocked circuit.

Reply:
I think it's still good to be able to control this so we can experiment in future abc versions, which might change their clocking behaviour, or to experiment to see how much proper clock handling is costing in case we're comparing to naive tools that don't do it.  It should be on by default though.

Reply:
Is this in regards to flow_type or use_new_latches_restoration_script?

Reply:
flow_type.  bool variables that are 0 or 1 are fine, but flow_type = 2 is mysterious.  flow_type = iterative_bb is more self-explanatory, if that is what flow_type = 2 means.  Or if flow_type = 2 means implement twice in two different flows and compare, then it should be renamed flows_to_compare = 2 or flow_type = implement_twice (or something like that).

Reply:
This has been fixed